src/actions/TransactionListActions.ts

```diff
@@ -92,19 +90,16 @@ const getAndMergeTransactions = async (state: RootState, dispatch: Dispatch, wal
     for (const tx of transactions) {
       // for each transaction, add some meta info
       const { date, time } = unixToLocaleDateTime(tx.date)
-      // @ts-expect-error
       if (!transactionIdMap[tx.txid]) {
         // if the transaction is not already in the list
-        // @ts-expect-error
-        transactionIdMap[tx.txid] = key
+        transactionIdMap[tx.txid] = true
         // @ts-expect-error
```

Comment:
I guess an `unfliteredIndex` is needed to satisfy the static type check, but what is the initial value for this I wonder.